clickhouse-mixin/rows.libsonnet

```diff
@@ -0,0 +1,47 @@
+local g = import './g.libsonnet';
+local panels = import './panels.libsonnet';
+
+{
+  new(this): {
+    replicaOperationsPanels: [
+      this.grafana.panels.interserverConnectionsPanel { gridPos+: { w: 12 } },
+      this.grafana.panels.replicaQueueSizePanel { gridPos+: { w: 12 } },
+      this.grafana.panels.replicaOperationsPanel { gridPos+: { w: 12 } },
+      this.grafana.panels.replicaReadOnlyPanel { gridPos+: { w: 12 } },
+    ],
```

Comment:
We found this format more appealing, as it allows to import separate rows and make them collapsed/uncollapsed later:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/rows.libsonnet#L5-L24
on the dashboard you need to add extra function:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/dashboards.libsonnet#L34

Reply:
I suppose this would change the dashboard's layout that would divert from the spec? The intention for this PR at least was to use the modular libs without changing the structure too much of the integration. 

Perhaps in a follow-up we could have actual row containers? Unless you feel like we should make this change in this PR or a quick follow-on PR?